### ISSUE TITLE
sword should be automatically awarded

### ISSUE THREAD

**Issue by stsp** [oitofelix/mininim#9]
If you cheat the guard and enter the second
level without taking the sword, you should be
awarded with the sword automatically. Otherwise
there is no way to go any further. Original game
did exactly that. There was no dead-end if you
enter level2 without a sword.

**Comment by oitofelix** [comment]
You are awarded with the sword if you enter level 2 in any way.  You have to press ENTER to draw the sword.  Please, refer to the manual.

**Comment by EndeavourAccuracy** [comment]
Recently, while playing MININIM, I skipped picking up the sword in level 1. In level 2, I thought I was not automatically awarded the sword. I did try pressing Ctrl (which brings up the sword in PoP2 for DOS) and random letters, mostly on the left side of the keyboard. I was wondering if not giving the prince his sword was a design choice. I decided to check the program's GitHub issues, including those closed, and found this ticket.

My opinion is that expecting - and if they don't, telling - users, who don't figure out themselves that Enter draws the sword, to read the program's manual is not the best approach. The post by @stsp points to a problem with MININIM, namely that it neither auto-draws the sword, nor tells its players in-game how to draw the sword. This is user-unfriendly. It would be similar to a company, that's shipping washing machines that have their power button hidden in a specific location, telling its customers that if they'd read the manual they knew how to turn on their devices.

Also, Enter isn't normally used for gameplay. It is a key to pick entries from menus and such. I don't think making the key Ctrl and then simply calling the program's behavior a design choice would be a solid solution to the problem either. Anyway, just my two-cents.
